build/source/netcdf/read_icond.f90

```diff
@@ -185,71 +186,79 @@ subroutine read_icond(iconFile,                      & ! name of initial conditi
  err = nf90_inq_dimid(ncID,"hru",dimID);               if(err/=nf90_noerr)then; message=trim(message)//'problem finding hru dimension/'//trim(nf90_strerror(err)); return; end if
  err = nf90_inquire_dimension(ncID,dimID,len=fileHRU); if(err/=nf90_noerr)then; message=trim(message)//'problem reading hru dimension/'//trim(nf90_strerror(err)); return; end if
 
- ! is there any snow in initial conditions file?
```

Comment:
Why did this come out? Will it work if there are no snow layers in the restart file?
When I was testing this, it failed if it tried to look for snow data where there was none.